mauro-api/src/main/groovy/org/maurodata/controller/folder/VersionedFolderController.groovy

```diff
@@ -143,7 +143,13 @@ class VersionedFolderController extends ModelController<Folder> implements Versi
     @Transactional
     @Put(Paths.VERSIONED_FOLDER_NEW_BRANCH_MODEL_VERSION)
     Folder createNewBranchModelVersion(UUID id, @Body @Nullable CreateNewVersionData createNewVersionData) {
-        super.createNewBranchModelVersion(id, createNewVersionData)
+        try {
+            super.createNewBranchModelVersion(id, createNewVersionData)
+        }
+        catch (Throwable th){
+            th.printStackTrace()
+            throw th
+        }
```

Comment:
could you remove the catch? we shouldn't have any errors here and they should be output to log anyway if they happen!